server/service/service.go

```diff
@@ -100,6 +100,9 @@ type Chassis struct {
 	BootloaderPasswordHash string
 	// The name of the fabric this chassis belongs to.
 	FabricName string
+	// Indicates whether enrollment will be skipped later in the install process.
+	// Some parts of boot config may be different depending on the value of this flag.
```

Comment:
Can you add a todo to either remove this, or refactor this so it makes more sense?

Reply:
Added a TODO and a bug to track it.